sd/etcd/client_test.go

```diff
@@ -0,0 +1,69 @@
+package etcd
+
+import (
+	"log"
+	"testing"
+	"time"
+
+	"golang.org/x/net/context"
+)
+
+func TestNewClient(t *testing.T) {
+	ClientOptions := ClientOptions{
+		Cert:                    "",
+		Key:                     "",
+		CaCert:                  "",
+		DialTimeout:             (2 * time.Second),
+		DialKeepAline:           (2 * time.Second),
+		HeaderTimeoutPerRequest: (2 * time.Second)}
+
+	client, err := NewClient(
+		context.Background(),
+		[]string{"http://localhost:2379"},
```

Comment:
How can this work? There's no build tag on this file, and I probably don't have an etcd running... ?